specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-10-01/migrate.json

```diff
@@ -2209,9 +2202,6 @@
             }
           }
         },
-        "x-ms-pageable": {
```

Comment:
this API is pageable.  ARM will return a nextLink if one of your regional endpoints doesn't respond in time or has a retryable error.

Reply:
applies to any collection GETs on tracked reosurces

Reply:
nextLinkName should be specified (and equal "nextLink") and the response in the subscription and resource group level collection GETs should return a nextLink property (the definition does not contain it right now).  Your RP doesn't control whether ARM returns a nextLink here or not and the SDK should be set up to follow it if it is returned.